### PR TITLE
Fix errors detected by the new checks script

### PR DESCRIPTION
**Pull request by lvoloshyn-sekoia**

### CODE REVIEW

File: Cisco/cisco-secure-firewall/tests/test_430002.json
Comment (squioc) [inline Cisco/cisco-secure-firewall/tests/test_430002.json:1]: Do you know why we lost extracted fields in that test?

File: Cisco/cisco-secure-firewall/tests/test_430003.json
Comment (squioc) [inline Cisco/cisco-secure-firewall/tests/test_430003.json:1]: Do you know why we lost extracted fields in that test?

File: Cisco/cisco-secure-firewall/tests/test_430005.json
Comment (squioc) [inline Cisco/cisco-secure-firewall/tests/test_430005.json:1]: Do you know why we lost extracted fields in that test?

File: utils/test_formats.py [hunk 1]
```diff
@@ -117,7 +117,7 @@ def test_intakes_produce_expected_messages(request, manager, intakes_root, test_
 
     expected = testcase["expected"]
 
-    if request.config.getoption("fix_expectations") and parsed != expected:
+    if True: #request.config.getoption("fix_expectations") and parsed != expected:
```
Comment (squioc) [inline utils/test_formats.py:120]: I'm not sure that you expect this code to land in the PR.

File: Acklio/acklio/_meta/smart-descriptions.json
Comment (squioc) [inline Acklio/acklio/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Azure/azure-linux/_meta/smart-descriptions.json
Comment (squioc) [inline Azure/azure-linux/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Azure/azure-windows/_meta/smart-descriptions.json
Comment (squioc) [inline Azure/azure-windows/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Checkpoint/checkpoint/_meta/smart-descriptions.json
Comment (squioc) [inline Checkpoint/checkpoint/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Cyberwatch/cyberwatch/_meta/smart-descriptions.json
Comment (squioc) [inline Cyberwatch/cyberwatch/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: F-Secure/fsecure-av/_meta/smart-descriptions.json
Comment (squioc) [inline F-Secure/fsecure-av/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: F5 Networks/f5-big-ip/_meta/smart-descriptions.json
Comment (squioc) [inline F5 Networks/f5-big-ip/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: HAProxy/haproxy/_meta/smart-descriptions.json
Comment (squioc) [inline HAProxy/haproxy/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: ISC DHCP/dhcpd/_meta/smart-descriptions.json
Comment (squioc) [inline ISC DHCP/dhcpd/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Ivanti/pulse-connect/_meta/smart-descriptions.json
Comment (squioc) [inline Ivanti/pulse-connect/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Linux/linux/_meta/smart-descriptions.json
Comment (squioc) [inline Linux/linux/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Log Insight Windows/log-insight-windows/_meta/smart-descriptions.json
Comment (squioc) [inline Log Insight Windows/log-insight-windows/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Panda Security/Aether/_meta/smart-descriptions.json
Comment (squioc) [inline Panda Security/Aether/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Panda Security/SIEM Feeder/_meta/smart-descriptions.json
Comment (squioc) [inline Panda Security/SIEM Feeder/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Postfix/postfix/_meta/smart-descriptions.json
Comment (squioc) [inline Postfix/postfix/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Pradeo/pradeo/_meta/smart-descriptions.json
Comment (squioc) [inline Pradeo/pradeo/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Tanium/tanium/_meta/smart-descriptions.json
Comment (squioc) [inline Tanium/tanium/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Tenable/alsid/_meta/smart-descriptions.json
Comment (squioc) [inline Tenable/alsid/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Umbrella/umbrella-dns/_meta/smart-descriptions.json
Comment (squioc) [inline Umbrella/umbrella-dns/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Umbrella/umbrella-ip/_meta/smart-descriptions.json
Comment (squioc) [inline Umbrella/umbrella-ip/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Umbrella/umbrella-proxy/_meta/smart-descriptions.json
Comment (squioc) [inline Umbrella/umbrella-proxy/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Wazuh/wazuh/_meta/smart-descriptions.json
Comment (squioc) [inline Wazuh/wazuh/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

File: Zeek/zeek/_meta/smart-descriptions.json
Comment (squioc) [inline Zeek/zeek/_meta/smart-descriptions.json:1]: I suggest adding an empty dict `{}` as content of the smart-descriptions files.

**Review by squioc — CHANGES_REQUESTED** [review]
Nice work.

I made some suggestions to improve the PR.

File: utils/test_formats.py (hunk 1 above)
Comment (lvoloshyn-sekoia) [reply utils/test_formats.py]: Yes, you're right. Thanks!

File: Umbrella/umbrella-dns/_meta/smart-descriptions.json
Comment (lvoloshyn-sekoia) [reply Umbrella/umbrella-dns/_meta/smart-descriptions.json:1]: I believe it should be [] instead, fixed

File: Zeek/zeek/_meta/smart-descriptions.json
Comment (lvoloshyn-sekoia) [reply Zeek/zeek/_meta/smart-descriptions.json:1]: I believe it should be [] instead, fixed

File: Wazuh/wazuh/_meta/smart-descriptions.json
Comment (lvoloshyn-sekoia) [reply Wazuh/wazuh/_meta/smart-descriptions.json:1]: I believe it should be [] instead, fixed

File: Umbrella/umbrella-proxy/_meta/smart-descriptions.json
Comment (lvoloshyn-sekoia) [reply Umbrella/umbrella-proxy/_meta/smart-descriptions.json:1]: I believe it should be [] instead, fixed

File: Umbrella/umbrella-ip/_meta/smart-descriptions.json
Comment (lvoloshyn-sekoia) [reply Umbrella/umbrella-ip/_meta/smart-descriptions.json:1]: I believe it should be [] instead, fixed

File: Cisco/cisco-secure-firewall/tests/test_430003.json
Comment (lvoloshyn-sekoia) [reply Cisco/cisco-secure-firewall/tests/test_430003.json:1]: Reverted, these shouldn't touched at all.

File: Cisco/cisco-secure-firewall/tests/test_430005.json
Comment (lvoloshyn-sekoia) [reply Cisco/cisco-secure-firewall/tests/test_430005.json:1]: Reverted, these shouldn't touched at all.

**Review by squioc — APPROVED** [review]

**Pull request merged**